sdk/src/kalix.ts

```diff
@@ -678,9 +679,7 @@ export class Kalix {
 
       const preStartSettings: PreStartSettings = {
         proxyHostname: proxyInfo.proxyHostname,
-        proxyPort:
-          this.proxyPort ??
-          (proxyInfo.proxyHostname == 'localhost' ? 9000 : 80),
+        proxyPort: proxyInfo.proxyPort,
```

Comment:
This might not be even needed. But I'm reluctant to use 9000 here as a hardcoded value that comes from Kalix proxy

Reply:
The code is correct, we should use whatever port the proxy send to us. Not 9000.